src/views/virtualmachinesinstance/details/VirtualMachinesInstancePageHeader.tsx

```diff
@@ -23,9 +23,7 @@ const VirtualMachinesInstancePageHeader: React.FC<VirtualMachinesInstancePageHea
       <VirtualMachineInstanceBreadcrumb />
       <span className={'co-m-pane__heading'}>
         <h1 className="co-resource-item__resource-name">
-          <span className={`co-m-resource-icon co-m-resource-icon--lg`}>
-            {t('VirtualMachineInstance')}
-          </span>
+          <span className={`co-m-resource-icon co-m-resource-icon--lg`}>{t('VMI')}</span>
```

Comment:
```suggestion
          <span className={`co-m-resource-icon co-m-resource-icon--lg`}>{t('VirtualMachineInstances')}</span>
```
?

Reply:
Before:
<img width="1274" alt="image" src="https://user-images.githubusercontent.com/61961469/170266120-6d11a8ed-8e2f-4391-bf42-9bf53c23e42d.png">
After:
<img width="1306" alt="image" src="https://user-images.githubusercontent.com/61961469/170266519-25c8214d-2616-4f71-a502-8ca749cdca78.png">
Also here, this is the change, What do you think?

Reply:
the kind label is not translatable, so it should just be 'VMI'
oyyy, it's also wrong here ...
https://github.com/kubevirt-ui/kubevirt-plugin/blob/main/src/views/virtualmachines/details/VirtualMachineNavPageTitle.tsx#L34

but it's more problematic, the kind label should not be hardcoded it should come from the object modal 
https://github.com/kubevirt-ui/kubevirt-api/blob/main/console/models/VirtualMachineInstanceModel.ts#L11